cves/CVE-2018-1336.yml

```diff
@@ -68,8 +70,8 @@ fixes:
    - commit:
      note:
 vccs:
-  - commit:
-    note:
+  - commit: ac4010b392a4406538ec5b29626f362c3e806c5b
+    note: SVN rev 1420654
```

Comment:
You can insert the SVN rev number to view the changes and also include the Log message in your notes. Here's the commit https://svn.apache.org/viewvc?view=revision&revision=1420654 .